pyinfra/operations/pipx.py

```diff
@@ -0,0 +1,100 @@
+"""
+Manage pipx (python) applications.
+"""
+
+from pyinfra import host
+from pyinfra.api import OperationError, operation
+from pyinfra.facts.pipx import PipxEnvironment, PipxPackages
+from pyinfra.facts.server import Path
+
+from .util.packaging import ensure_packages
+
+
+@operation
+def packages(
+    packages=None,
+    present=True,
+    latest=False,
+    requirements=None,
+    pipx="pipx",
+    extra_install_args=None,
+):
+    """
+    Install/remove/update pipx packages.
+
+    + packages: list of packages to ensure
+    + present: whether the packages should be installed
+    + latest: whether to upgrade packages without a specified version
+    + pipx: name or path of the pipx binary to use
+    + extra_install_args: additional arguments to the pipx install command
+
+    Versions:
+        Package versions can be pinned like pip: ``<pkg>==<version>``.
+
+    **Example:**
+
+    .. code:: python
+
+        pipx.packages(
+            name="Install ",
+            packages=["pyinfra"],
+        )
+    """
+
+    # We should always use the --force flag because
+    # if install it's called, that meen the version missmatch
+    # so we need the --force flag to get the good version installed
+    install_command = [pipx, "install", "--force"]
```

Comment:
I think force should be a keyword argument for the operation, rather than added by default. We should be hitting the upgrade command if a package is already installed?